test/controllers/articles_controller_test.rb

```diff
@@ -0,0 +1,7 @@
+require 'test_helper'
+
+class ArticlesControllerTest < ActionDispatch::IntegrationTest
+  # test "the truth" do
```

Comment:
What do we need any of these "articles" resources for? I think we could delete `articles_controller.rb`, `articles_helper.rb`, `articles.js`, etc. At the very least, we should add them all in a separate commit so we have one commit for "Add welcome controller and view" and one for "add articles controller and view"

Reply:
Yes, they are not required for the issue. It was just a part of the getting started manual. I'll delete them and push again.

Reply:
We can also delete the controller test file, the resource in `routes.rb`, the view file, and the scss file